model/src/gateway/payload/thread_members_update.rs

```diff
@@ -0,0 +1,17 @@
+use crate::{
+    channel::ThreadMember,
+    id::{ChannelId, GuildId, UserId},
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct ThreadMembersUpdate {
+    pub id: ChannelId,
+    pub guild_id: GuildId,
+    /// Capped at 50.
```

Comment:
For consistency with the other comments:
```suggestion
    /// Max value of 50.
```